src/commands/createFunction/createFunction.ts

```diff
@@ -64,9 +65,10 @@ export async function createFunctionInternal(context: IActionContext, options: a
     }
 
     const { language, languageModel, version } = await verifyInitForVSCode(context, projectPath, options.language, /* TODO: languageModel: */ undefined, options.version);
+    const hasDurableStorage: boolean = await durableUtils.verifyHasDurableStorage(language, projectPath);
```

Comment:
I figured it was easier to manage/more flexible to just add this onto the `wizardContext` at the beginning of the create process since it can already be known so I don't have to worry about passing/managing a separate variable later on.